### PR TITLE
Update installation docs from pip to pip3

### PR DESCRIPTION
**Pull request by sayanchowdhury**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ A versatile IRC bouncer.
   it from your distro's repositories
 - Create a Python3 virtualenv: ``mkvirtualenv ircb --no-site-packager -p /usr/bin/python3``
 - Switch to your virtualenv: ``workon ircb``
-- Install dependencies: ``pip install -r requirements.txt``
+- Install dependencies: ``pip install -r requirements.txt``. Incase the
+  installation is not done inside an environment, then install the dependencies
```
Comment (pypingou) [inline README.md]: a virtual environment?

Comment (rtnpro) [inline README.md]: Yes.

Comment (rtnpro) [inline README.md]: However, I think it's better to be more explicit and use `pip3` and `python3` instead of `pip` and `python`.

Comment (sayanchowdhury) [inline README.md]: Let me add some more changes as it will be python3 instead of python.

Comment (rtnpro) [inline README.md]: @sayanchowdhury could you do the above change and remove `virtualenv` related instructions?

Comment (sayanchowdhury) [inline README.md]: Okay.

**Pull request merged**